test/time_test.py

```diff
@@ -4,6 +4,15 @@
 
 Clock = pygame.time.Clock
 
+SDL1 = pygame.get_sdl_version()[0] < 2
+
+# Because timing based tests are twitchy, just test stuff again if they fail,
+# but only upto NUM_TRIES
+NUM_TRIES = 3
+
+# here, delta represents allowable error ratio
+def _is_almost_equal(a, b, delta):
+    return b * (1 - delta) <= a <= b * (1 + delta)
 
 class ClockTypeTest(unittest.TestCase):
     __tags__ = ['timing']
```

Comment:
also one line break after this line, before the first function definition.

Reply:
Done :)